clang/docs/checkedc/3C/README.md

```diff
@@ -0,0 +1,64 @@
+# 3C: Semi-automated conversion of C code to Checked C
+
+The 3C (**C**hecked-**C**-**C**onvert) software facilitates conversion of existing C code to Checked C by automatically inferring as many Checked C annotations as possible and guiding the developer through the process of assigning the remaining ones.  3C aims to provide the first feasible way for organizations with large legacy C codebases (that they don't want to drop everything to rewrite in a better language) to comprehensively verify their code's spatial memory safety.
+
+## What 3C users should know about the development process
+
+Development of 3C is led by [Correct Computation, Inc.](https://correctcomputation.com/) (CCI) in the https://github.com/correctcomputation/checkedc-clang repository, which is a fork of Microsoft's Checked C repository (https://github.com/microsoft/checkedc-clang).  Issues and pull requests related to 3C should be submitted to the CCI repository; see [CONTRIBUTING.md](CONTRIBUTING.md) for more information.  Changes are merged frequently from Microsoft's repository to CCI's and less frequently in the opposite direction.  While some automated tests of 3C are run in Microsoft's repository, the coverage of these tests is currently mediocre, so changes to Microsoft's repository may break some functionality of its copy of 3C.  Thus, users of 3C can choose either CCI's repository (for the latest 3C with a somewhat older Checked C) or Microsoft's (for the latest Checked C with 3C that is significantly older and possibly broken).  This workflow may change in the future.
+
+As of November 2020, 3C is pre-alpha quality and we are just starting to establish its public presence and processes.  Feel free to go ahead and file [issues](https://github.com/correctcomputation/checkedc-clang/issues) to help inform us of what users want; if we find that we're deluged with issues we're unprepared to address at this stage of development, we'll establish a different policy.  We strive to promptly fix any issues that block basic usage of 3C in our [supported environments](INSTALL.md#supported-environments), but beyond that, we have our own plans that are rapidly evolving and you should not expect that issues will be resolved in any given timeframe.
+
+CCI is also working on a proprietary extension of 3C called 5C ("**C**orrect **C**omputation's **C**hecked-**C**-**C**onvert").  Our current plan is that 3C will contain the core inference logic, while 5C will add features to enhance developer productivity.  If you'd like more information about 5C, please contact us at info@correctcomputation.com.
+
+## Functionality of 3C
+
+(TODO)
+
+(TODO: Is anything from https://github.com/correctcomputation/checkedc-clang/wiki/Checked-C-Convert#running-checked-c-convert worth including?)
```

Comment:
Yes. Basic usage information should go somewhere. 
- How to run on a single file, with output going to `stdout`
- Running with multiple files requires simply passing them all on the command line, along with `-output-postfix` set; e.g., `3c -output-postfix=checked a.c b.c` will output files `a.checked.c` and `b.checked.c`.
- Ensuring that source files are processed properly (e.g., with the right preprocessor environment) requires a few steps. (This is where we have to talk about using BEAR etc. Maybe Kyle or Shilpa could find the right things to say here?)

Reply:
@mwhicks1 Can you please open new threads for anything here that is still relevant and then resolve this one?